### ISSUE TITLE
spp_demo  crashed

### ISSUE THREAD

**Issue by VirtualEarth** [ShaoqingRen/SPP_net#34]
I ran "spp_demo" under following conditions:
           GTX760   2G memory
           GPU mode
          "varargin" of "caffe_anysize_test.m"  is 1800*1200*3
After executing "[result] = caffe('forward', varargin);", matlab crashed. Does this problem due to insufficient GPU memory (2G）?

**Comment by ShaoqingRen** [comment]
@VirtualEarth 

Yes, please modify Line55 in spp_features_convX.m. So that for large image, the computation will be hold on CPU.

**Comment by andygjq** [comment]
Dear:
         I didn't find the right dataset for training , when I run the file "Script_spp_voc.m", I haven't the file of "data\selective_search_data\voc_2007_trainval.mat", where should I download these?

**Comment by VirtualEarth** [comment]
I downloaded   the voc2007 dataset and combined trainval with test dataset. 
http://pascallin.ecs.soton.ac.uk/challenges/VOC/voc2007/

**Comment by andygjq** [comment]
however, I run the  "Script_spp_voc.m", 
It show that I was wrong on lack the file  "data\selective_search_data\voc_2007_trainval.mat".
Is I can skip it ?

**Comment by VirtualEarth** [comment]
You should download SPP_net_release1_VOC2007_selective_search_data.zip.
https://www.dropbox.com/sh/394evpav3b2swif/AAAstTWUztnzsFUKb0ZvQtQRa?dl=0

**Comment by andygjq** [comment]
I am very sorry for making such a request , I hope you can send the file to my email  gujianquan123@gmail.com
I have try several browers to download the file, but it failed.
Best wish!

**Comment by VirtualEarth** [comment]
@andygjq 
hehe , maybe you can  download files here.
http://pan.baidu.com/s/1mgKaUSS

**Comment by andygjq** [comment]
Thanks ! I can download the files now.
